tornado_http2/connection.py

```diff
@@ -34,7 +34,8 @@ def __init__(self, stream, is_client, params=None, context=None):
         self.context = context
         self._initial_settings_written = Future()
         self._serving_future = None
-        self._settings = {}
+        self._settings = {setting.code : setting.default for setting in
```

Comment:
I don't think we want to copy all the settings here; it's kind of expensive (we have a few places in tornado that use a similar pattern that show up a lot in profiles). It's also unnecessary as long as all accesses go through `Connection.setting` which incorporates the default.

Reply:
Ah yes, didn't see that the property used the defaults, I've removed this from the diff